cmd/kubestellar-operator/main.go

```diff
@@ -60,10 +60,12 @@ func main() {
 	var probeAddr string
 	var wdsName string
 	var wdsLabel string
+	var allowedGroupsString string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&wdsName, "wds-name", "", "name of the workload description space to connect to")
 	flag.StringVar(&wdsLabel, "wds-label", "", "label of the workload description space to connect to")
+	flag.StringVar(&allowedGroupsString, "api-groups", "", "list of allowed api groups, comma separated. If not specified, all api groups are allowed.")
```

Comment:
FYI:
`-h` prints the default value, right? I would make the description here say "Empty string means all API groups are allowed".

Reply:
done